### PR TITLE
HN 11 show flagged content

### PR DESCRIPTION
**Pull request by BakkerTom**
In this pull request, I've included the ability to filter flagged content in the web editor and see reasons why posts are flagged among other changes...

- UI Improvements
- Tweets are now embedded with the Twitter API
- The ```FilterBar``` component handles the toggling of multiple filters
- The ```Feed``` is now able to accept filters and act accordingly
- A new QueryParameter is added to the admin PostController which returns only flagged content when true
---
Before submitting your pull request, consider the following: Does your code...

- [x] Implement functionality
- [x] Contain comments/documentation
- [x] Build without errors
- [x] Contain and pass unit testing
- [x] Conform to the Style Guide
- [x] Have no merge conflicts with ```master```

When you've checked all of the boxes above your code is up for review. If approved it can be merged with the ```master``` branch.

If you want feedback on your work before having finished your code, add ```[WIP]``` to the title of your PR.

### CODE REVIEW

File: api/src/main/java/nl/fhict/happynews/api/controller/admin/AdminPostController.java [hunk 1]
```diff
@@ -52,13 +55,21 @@ protected void initPostBinder(WebDataBinder binder) {
...
      *
      * @param pageable the page and page size
+     * @param isFiltered only displays flagged posts if true, false by default
      * @return A Page with Post information
      */
     @ApiOperation("Get all posts in a paginated format")
     @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
-    public Page<Post> getAllByPage(Pageable pageable) {
+    public Page<Post> getAllByPage(Pageable pageable,
+                                   @RequestParam(required = false, defaultValue = "false") Boolean isFiltered) {
```
Comment (TvanBronswijk) [inline api/src/main/java/nl/fhict/happynews/api/controller/admin/AdminPostController.java]: `isFiltered` should probably be `filter`.

File: web/src/main/client/src/components/feed/Feed.js [hunk 2]
```diff
@@ -4,27 +4,37 @@ import ReactPaginate from 'react-paginate';
...
   }
 
-  //loadItems load
+  /**
+   * load the items via the Happy News API
+   * @param {int} pageNumber 
+   */
   loadItems(pageNumber){
-    const url = `https://happynews-api.svendubbeld.nl/admin/posts?page=${pageNumber}&size=${PAGE_SIZE}`;
+    const url = `https://happynews-api.svendubbeld.nl/admin/posts?page=${pageNumber}&size=${PAGE_SIZE}&isFiltered=${this.state.filtered}`;
```
Comment (SvenDub) [inline web/src/main/client/src/components/feed/Feed.js]: I think `filtered` would be a better name for the parameter.

File: api/src/main/java/nl/fhict/happynews/api/controller/admin/AdminPostController.java (hunk 1 above)
Comment (BakkerTom) [inline api/src/main/java/nl/fhict/happynews/api/controller/admin/AdminPostController.java]: WINNER WINNER CHICKEN DINNER

File: web/src/main/client/src/components/feed/Feed.js (hunk 2 above)
Comment (BakkerTom) [inline web/src/main/client/src/components/feed/Feed.js]: We are going with 'filter'!

**Pull request merged**